### PR TITLE
monterey-cache-cleaner: transition to northern-softworks-cache-cleaner

### PR DESCRIPTION
**Pull request by p-linnane**
Updating as in previous years.
Big Sur -> Monterey: https://github.com/Homebrew/homebrew-cask/pull/113881
Catalina -> Big Sur: https://github.com/Homebrew/homebrew-cask/pull/92907

### CODE REVIEW

**Comment by SMillerDev** [comment]
Could you add a zap? And maybe we should name it differently?

File: Casks/ventura-cache-cleaner.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask "ventura-cache-cleaner" do
+  version "18.0"
+  sha256 :no_check
+
+  url "https://www.northernsoftworks.com/downloads/nscc.dmg"
+  name "Monterey Cache Cleaner"
+  name "MCC"
```
Comment (carlocab) [inline Casks/ventura-cache-cleaner.rb:7]:
```suggestion
  name "Ventura Cache Cleaner"
  name "VCC"
```
?

Comment (p-linnane) [reply Casks/ventura-cache-cleaner.rb]: That was dumb. Thanks!

**Comment by SMillerDev** [comment]
`northern-softworks-cache-cleaner` sounds good to me, it might be long but most people just copy it anyway.

**Comment by SMillerDev** [comment]
No zap needed?

**Comment by p-linnane** [comment]
Forgot with the whole renaming thing. Added the zap and a comment explaining the changes for ever major macOS version.

**Review by miccal — APPROVED** [review]

**Pull request merged**